config/initializers/new_framework_defaults_5_2.rb

```diff
@@ -0,0 +1,38 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 5.2 upgrade.
+#
+# Once upgraded flip defaults one by one to migrate to the new default.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+
+# Make Active Record use stable #cache_key alongside new #cache_version method.
+# This is needed for recyclable cache keys.
+# Rails.application.config.active_record.cache_versioning = true
+
+# Use AES-256-GCM authenticated encryption for encrypted cookies.
+# Also, embed cookie expiry in signed or encrypted cookies for increased security.
+#
+# This option is not backwards compatible with earlier Rails versions.
+# It's best enabled when your entire app is migrated and stable on 5.2.
+#
+# Existing cookies will be converted on read then written with the new scheme.
+# Rails.application.config.action_dispatch.use_authenticated_cookie_encryption = true
```

Comment:
Looks like we should set this to false to start, until we're confident that the upgrade is stable. Otherwise, we won't be able to roll back without possibly invalidating all user cookies.

Reply:
Yeah, I don't know enough about our clients and how long we want their sessions to last...though it feels like forcing folks to re-login wouldn't be a bad thing to do with this upgrade?
In any case, I set this to false for now.

Reply:
The risk is that during clinic hours, if a nurse is logged out and has to sign in again to continue treating patients, it could be a very disruptive experience. Since we have the option of avoiding this friction, let's do it 👌

Reply:
For what it's worth, nurses don't deal with cookies. That should only affect the web side of things for CVHOs, and it looks like Rails auto-migrates them.

Let's keep this as `false` for now which allows us to roll back, but change it once we're confident the app works.